src/lib/pay-request/api_utils/ledger.js

```diff
@@ -47,7 +47,25 @@ const ledgerMethods = function ledgerMethods(instance) {
       .then(utilExtractData)
   }
 
-  return { transaction, transactions, events }
+  const statistics = function statistics(account, fromDate, toDate, override) {
+    const params = {
+      ...account && { account_id: account },
+      ...fromDate && { from_date: fromDate },
```

Comment:
Any params that are required (will always be filled in) can just be put directly on the object. 

```js 
const params = { 
  from_date: fromDate, 
  to_date: toDate,
  ...account && { acccount_id: account }
}
```

Reply:
Done